### PR TITLE
add project creator permission

### PR DESCRIPTION
**Pull request by mirnawong1**
## Description & motivation

Added project creator permissions in enterprise docs per slack threads below and users asking:
https://dbt-labs.slack.com/archives/C02SRNY2EQ4/p1666795228671769
https://dbt-labs.slack.com/archives/C03C405UE64/p1666799829783969

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/collaborate/manage-access/enterprise-permissions.md [hunk 1]
```diff
@@ -44,6 +44,29 @@ Account Admins have unrestricted access to dbt Cloud accounts. Users with Accoun
...
+- Create, delete, and modify Repositories
+- Create, delete, and modify Connections
+- Create, delete, and modify Environments
+- Create, delete, and modify Jobs
+- Use the IDE
+- Run and cancel jobs
+
+<!-- not sure if the below is correct -->
+- View Groups
+- Create, delete, and modify Group Memberships
```
Comment (ajaythomas) [inline website/docs/docs/collaborate/manage-access/enterprise-permissions.md]: You cannot do this as a project creator. I dont see the `groups_write` perm [there](https://github.com/dbt-labs/dbt-cloud/blob/master/sinter/models/permissions/sets/account/project_creator.py#L20).

File: website/docs/docs/collaborate/manage-access/enterprise-permissions.md [hunk 2]
```diff
@@ -44,6 +44,29 @@ Account Admins have unrestricted access to dbt Cloud accounts. Users with Accoun
...
+- Manage notification settings
```
Comment (ajaythomas) [inline website/docs/docs/collaborate/manage-access/enterprise-permissions.md]: I believe you can only [view notifications](https://github.com/dbt-labs/dbt-cloud/blob/master/sinter/services/notifications.py#L107). [Not manage them](https://github.com/dbt-labs/dbt-cloud/blob/master/sinter/services/notifications.py#L68).

@parose @carusocv @gcs272 can you confirm?

Comment (parose) [reply website/docs/docs/collaborate/manage-access/enterprise-permissions.md]: That's my understanding - it looks like you need [account_settings_write](https://github.com/dbt-labs/dbt-cloud/blob/828ead742448e249436197b03d5ae884864f3b81/sinter/services/notifications.py#L68) to manage notifications, which the project creator permission set doesn't have

Comment (mirnawong1) [reply website/docs/docs/collaborate/manage-access/enterprise-permissions.md]: hi @ajaythomas and @parose , many thanks and i've made those edits now - can I ask you to have a look and review/approve for technical accuracy? i will then ask for a Docs team review once this is done and get this up on our docs site. Many thanks!

**Review by ajaythomas — DISMISSED** [review]

File: website/docs/docs/collaborate/manage-access/enterprise-permissions.md [hunk 3]
```diff
@@ -44,6 +44,26 @@ Account Admins have unrestricted access to dbt Cloud accounts. Users with Accoun
 - Use the IDE
 - Run and cancel jobs
 
+### Project Creator
+- **Has permissions on:** Authorized projects, account-level settings
+- **License restrictions:** must have a developer license
+
+Project Creators have write and read only access to dbt Cloud accounts, but do not have permissions to modify SSO settings and account integrations.
```
Comment (matthewshaver) [inline website/docs/docs/collaborate/manage-access/enterprise-permissions.md]:
```suggestion
Project Creators have write and read-only access to dbt Cloud accounts, but do not have the permissions required to modify SSO settings and account integrations. 
```

File: website/docs/docs/collaborate/manage-access/enterprise-permissions.md [hunk 4]
```diff
@@ -44,6 +44,26 @@ Account Admins have unrestricted access to dbt Cloud accounts. Users with Accoun
...
+- View and modify project users
+- Create, delete and modify all projects in an account
+- Create, delete, and modify Repositories
+- Create, delete, and modify Connections
+- Create, delete, and modify Environments
+- Create, delete, and modify Jobs
+- Use the IDE
+- Run and cancel jobs
+- View Groups
+- View notification settings
```
Comment (matthewshaver) [inline website/docs/docs/collaborate/manage-access/enterprise-permissions.md]: For my own clarification, how are we choosing what to capitalize after the first word?  Are the capitalized words all menu items or feature selections in the product?  If that's the case should "notification settings" be capitalized?

Comment (mirnawong1) [reply website/docs/docs/collaborate/manage-access/enterprise-permissions.md]: ah yes good callout @matthewshaver ! any UI items were capitalized to match the UI but i noticed the other permissions didn't capitalize 'Notification Settings' so i wasn't sure if it should be lowercase. But I've double-checked the UI and have changed it and also changed the other permissions too.

**Review by matthewshaver — APPROVED** [review]

**Comment by mirnawong1** [comment]
thank you so much @matthewshaver !

**Pull request merged**